Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -492,6 +492,10 @@
 #include "CoreIPCAuditToken.h"
 #endif
 
+#if __has_include(<WebKitAdditions/WebPreferencesDefaultValuesAdditions.h>)
+#include <WebKitAdditions/WebPreferencesDefaultValuesAdditions.h>
```

Comment:
Nit - we should rename this to reflect that it's no longer inside `WebPreferencesDefaultValues`, but this is okay for now.

Reply:
Thanks for reviewing!